src/servers/src/http/event.rs

```diff
@@ -569,9 +572,13 @@ pub(crate) async fn ingest_logs_inner(
                 .get_table(&request.table, &query_ctx)
                 .await
                 .context(CatalogSnafu)?;
-            pipeline::identity_pipeline(request.values, table)
-                .context(PipelineTransformSnafu)
-                .context(PipelineSnafu)?
+            pipeline::identity_pipeline(
+                request.values,
+                table,
+                GreptimeIdentityPipelineParams::from_headers(&headers),
```

Comment:
How about parsing the headers to params here in `event.rs` so that the `pipeline` crate doesn't have to depend on the `axum` crate.

Reply:
I can refactor this in #5409

Reply:
make sense.